### DISCUSSION TITLE
gmail lecture videos

### DISCUSSION THREAD

**Discussion by hawks12345** [aclap-dev/video-downloadhelper#1151] [Help and Q&A]
### Your question

Hello everyone! I just purchased the premium. I noticed that i couldn't download the unencrypted videos sent to me via email by my lecturer. The icon is still grayed even if the video is already playing. I hope there is some way to fix that coz it really is for my studies. Thanks in advance!

### Addon & CoApp version and details

Hello everyone! I just purchased the premium. I noticed that i couldn't download the unencrypted videos sent to me via email by my lecturer. The icon is still grayed even if the video is already playing. I hope there is some way to fix that coz it really is for my studies. Thanks in advance!

**Comment by Mattmjs** [comment]
What do you mean ? Where are the videos embedded ? I know there's an issue on Google drive video.
